src/app/folder/folderlist.component.html

```diff
@@ -50,6 +50,10 @@
                 <mat-icon color="primary">border_color</mat-icon>
                 <span>Rename</span>
               </button>
+              <button mat-menu-item (click)="addSubFolder(node)">
+                <mat-icon color="primary">create_new_folder</mat-icon>
+                <span>New Folder</span>
```

Comment:
Perhaps we can make that “New Subfolder” to make it distinct? @geir, @davidbowdley?

The resulting popup still says “New folder”, but also “under XXX”

Reply:
Nice work @kvviecien, and agree @tadzik -- "New Subfolder" would be technically correct while confirming to the user that they are not creating a folder at the root level.